### DISCUSSION TITLE
Referencing Building footprint data from Microsoft Global Building footprint data and street network data from OSM to classify urban fabric in UTs.

### DISCUSSION THREAD

**Discussion by Chirag11011100** [pysal/momepy#434] [Q&A] [answered]
How can I use the Microsoft global building footprint data which is downloaded in a custom map in QGIS and street network data which is taken from OSM to classify urban fabric into UTs for Indian cities? @martinfleis ?

**Comment by martinfleis** [comment, marked as answer]
I believe this is already answered at https://github.com/pysal/momepy/discussions/440.

**Reply by r-li-lu-ya** [reply]
Hello,i have a question——How to classify types of urban residential buildings using k-meas clustering

**Reply by martinfleis** [reply]
That is a very wide question. You can start here http://docs.momepy.org/en/stable/examples/clustering.html and look at the Youtube video linked there.
